src/main/java/ToDo.java

```diff
@@ -0,0 +1,21 @@
+public class ToDo extends Task{
+    ToDo(String name){
```

Comment:
Consider adding a space before the opening braces. I have noticed the same issue in several other places too.